ucfcs/GrowthPlus
# Team/level one rm

This PR implements lesson 1 roadmap one. 

To test:
Do a general test of the functionality of the app. 
Go to roadmap one and tab tile one, it should direct you to lesson one. 
Go through the lessons, make sure nothing breaks.
